fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFragment.kt

```diff
@@ -40,14 +40,14 @@ class ReviewQualityCheckFragment : BottomSheetDialogFragment() {
     private var behavior: BottomSheetBehavior<View>? = null
     private val bottomSheetStateFeature =
         ViewBoundFeatureWrapper<ReviewQualityCheckBottomSheetStateFeature>()
-    private val store by lazy {
+    private val store by lazyStore { scope ->
```

Comment:
As we're using this as an example on how to use the scope, let's make it clear which scope it is that we are replacing the `lifecycleScope` with.
```suggestion
    private val store by lazyStore { viewModelScope ->
```

Reply:
Since we have abstracted away `ViewModel` from Store and features, it made sense to not reference it. I don't have a strong opinion here, but it's a bit confusing for the caller to reference `viewModelScope` when using Store, thoughts?

Reply:
We're still using the provider to get the ViewModel whose scope we are using implicitly. It seems more clear to give the caller some clarity whose scope this is when they decide to use it.

Since we will use this PR as reference when you share it to the team in our mailing list and groups, so I heed on the side of clarity for that case. i.e. I don't need to look at the implementation, to follow the usage and know which scope this is.

Reply:
Sure! I am biased to putting `Provider` in there too because it gives you a bit more context that this will give you a `Store`. Let's do it in a separate change though?

Reply:
yeah, unrelated to this change. Again, no strong opinion here, discussing more from the perspective of increasing clarity.